.github/workflows/swift.yml

```diff
@@ -7,51 +7,16 @@ on:
     branches: [main]
 
 jobs:
-  catalina-swift5_2:
-    runs-on: macos-10.15
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build on macOS 10.15 with Swift 5.2
-        run: |
-          sudo xcode-select --switch /Applications/Xcode_11.7.app/Contents/Developer
-          swift test -c release --enable-test-discovery
-          swift build -c release
-          brew bundle
-          cd TestApp && ../.build/release/carton test
-          ../.build/release/carton test --environment defaultBrowser
-          ../.build/release/carton bundle
-        # the token is required to get around GitHub API limits when downloading the toolchain
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-  bigsur-swift5_2:
-    runs-on: macos-11.0
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build on macOS 11.0 with Swift 5.2
-        run: |
-          sudo xcode-select --switch /Applications/Xcode_11.7.app/Contents/Developer
-          swift test -c release --enable-test-discovery
-          swift build -c release
-          brew bundle
-          cd TestApp && ../.build/release/carton test
-          ../.build/release/carton test --environment defaultBrowser
-          ../.build/release/carton bundle
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
   catalina-swift5_3:
     runs-on: macos-10.15
 
     steps:
       - uses: actions/checkout@v2
       - name: Build on macOS 10.15 with Swift 5.3
         run: |
-          sudo xcode-select --switch /Applications/Xcode_12.app/Contents/Developer
-          swift test -c release --enable-test-discovery
+          sudo xcode-select --switch /Applications/Xcode_12.3.app/Contents/Developer
           swift build -c release
+          swift test -c release --enable-test-discovery
           brew bundle
           cd TestApp && ../.build/release/carton test
           ../.build/release/carton test --environment defaultBrowser
```

Comment:
This can be removed. It's included in the CI Tests now.

If we are going to build before tests it should be

```
swift build --build-tests --enable-test-discovery
swift test --enable-test-discovery
```